addons/mail/i18n/mail.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-10 10:22+0000\n"
-"PO-Revision-Date: 2022-10-10 10:22+0000\n"
+"POT-Creation-Date: 2022-10-26 13:41+0000\n"
+"PO-Revision-Date: 2022-10-26 13:41+0000\n"
```

Comment:
We usually don't update these meta data in the diff as they are not really useful and when updated they are annoying to manage when there are conflicts between PRs.

Reply:
This wasn't updated either.